lib/charms/hydra/v0/oauth.py

```diff
@@ -401,8 +430,8 @@ def is_client_created(self, relation_id: Optional[int] = None) -> bool:
             return None
 
         return (
-            "client_id" in relation.data[relation.app]
-            and "client_secret_id" in relation.data[relation.app]
+            relation.data[relation.app]["client_id"]
+            and relation.data[relation.app]["client_secret_id"]
```

Comment:
How do we know that these keys are in the relation and will not throw a `KeyError`?